Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG IMAGE_VERSION
 
 RUN set -eux \
     && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get update \
+    && apt-get upgrade;apt-get update \
```

Comment:
I wish we have better options for this. I also can't think of better alternatives.

Reply:
I am not sure if auto builds will always fetch the latest base image?